pkg/module_manager/scheduler/scheduler.go

```diff
@@ -414,10 +519,12 @@ func (s *Scheduler) GetGraphState(logLabels map[string]string) ( /*enabled modul
 	}
 
 	if len(s.errList) > 0 {
-		return s.getEnabledModuleNames(), nil, fmt.Errorf("couldn't recalculate graph: %s", strings.Join(s.errList, ","))
+		return s.getEnabledModuleNames(), nil, nil, fmt.Errorf("couldn't recalculate graph: %s", strings.Join(s.errList, ","))
 	}
 
-	return s.getEnabledModuleNames(), s.gleanGraphDiff(), nil
+	enabledModulesByOrder, err := s.getEnabledModuleNamesByOrder()
+
+	return s.getEnabledModuleNames(), enabledModulesByOrder, s.gleanGraphDiff(), err
```

Comment:
Do we really need to fetch all modules, if `getEnabledModuleNamesByOrder` have returned an error? Will we use them?

Reply:
nope. should be fine w/o it